### ISSUE TITLE
fastlane match enterprise giving an error for new generated certificate "Could not find the newly generated certificate installed" 

### ISSUE THREAD

**Issue by katanaBoy** [fastlane/fastlane#17355]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I can't find newly generated certificate and profile when I used "fastlane match enterprise"
though I've set MATCH_KEYCHAIN_NAME=flipdish.keychain I don't see the private key and cert generated in that key chain. What else I see in the log is that it tries to search login.keychain-db dough flipdish.keychain is set. 

Let me know if you want to add something more. Thank you
##### Command executed

fastlane match enterprise --verbose

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>
export MATCH_GIT_BASIC_AUTHORIZATION=auth_token
export MATCH_PASSWORD=password
export MATCH_KEYCHAIN_NAME=flipdish.keychain
export MATCH_KEYCHAIN_PASSWORD=keychainPass
export FASTLANE_SESSION = appleToken
fastlane match enterprise --verbose
[✔] 🚀 
[17:47:49]: fastlane detected a Gemfile in the current directory
[17:47:49]: However, it seems like you didn't use `bundle exec`
[17:47:49]: To launch fastlane faster, please use
[17:47:49]: 
[17:47:49]: $ bundle exec fastlane match enterprise --verbose
[17:47:49]: 
[17:47:49]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
INFO [2020-09-30 17:48:02.91]: Successfully loaded '/Users/macbookair/Desktop/Code/ios/fastlane/Matchfile' 📄

+--------------+-------------------------------------------------+
|          Detected Values from './fastlane/Matchfile'           |
+--------------+-------------------------------------------------+
| git_url      | https://github.com/flipdishbytes/appsigning.git |
| storage_mode | git                                             |
| type         | development                                     |
+--------------+-------------------------------------------------+

+--------------------------------+-------------------------------------------------+
|                            Summary for match 2.161.0                             |
+--------------------------------+-------------------------------------------------+
| verbose                        | true                                            |
| type                           | enterprise                                      |
| readonly                       | false                                           |
| generate_apple_certs           | true                                            |
| skip_provisioning_profiles     | false                                           |
| storage_mode                   | git                                             |
| git_url                        | https://github.com/flipdishbytes/appsigning.git |
| git_branch                     | master                                          |
| shallow_clone                  | false                                           |
| clone_branch_directly          | false                                           |
| git_basic_authorization        | ********                                        |
| keychain_name                  | flipdish.keychain                               |
| keychain_password              | ********                                        |
| force                          | false                                           |
| force_for_new_devices          | false                                           |
| skip_confirmation              | false                                           |
| skip_docs                      | false                                           |
| platform                       | ios                                             |
| derive_catalyst_app_identifier | false                                           |
| fail_on_name_taken             | false                                           |
+--------------------------------+-------------------------------------------------+

WARN [2020-09-30 17:48:02.93]: To not be asked about this value, you can specify it using 'username'
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/highline-1.7.10/lib/highline.rb:624: warning: Using the last argument as keyword parameters is deprecated
 [2020-09-30 17:48:02.93]: Your Apple ID Username: acc@flipdish.ie
INFO [2020-09-30 17:48:22.00]: Cloning remote git repo...
INFO [2020-09-30 17:48:22.00]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
INFO [2020-09-30 17:48:22.00]: $ git clone https://github.com/flipdishbytes/appsigning.git /var/folders/t7/pjy501d53hzg38d68vdnyyrh0000gn/T/d20200930-11040-1548rs4 -c http.extraheader='Authorization: Basic auth_token'
INFO [2020-09-30 17:48:22.14]: ▸ Cloning into '/var/folders/t7/pjy501d53hzg38d68vdnyyrh0000gn/T/d20200930-11040-1548rs4'...
INFO [2020-09-30 17:48:23.29]: ▸ remote: Enumerating objects: 11, done.
INFO [2020-09-30 17:48:23.29]: ▸ remote: Counting objects: 100% (11/11), done.
INFO [2020-09-30 17:48:23.34]: ▸ remote: Compressing objects: 100% (7/7), done.
INFO [2020-09-30 17:48:23.34]: ▸ remote: Total 11 (delta 0), reused 8 (delta 0), pack-reused 0
INFO [2020-09-30 17:48:23.37]: ▸ Unpacking objects: 100% (11/11), done.
INFO [2020-09-30 17:48:23.41]: $ git --no-pager branch --list origin/master --no-color -r
INFO [2020-09-30 17:48:23.43]: ▸   origin/master
INFO [2020-09-30 17:48:23.43]: Checking out branch master...
INFO [2020-09-30 17:48:23.43]: $ git checkout master
INFO [2020-09-30 17:48:23.46]: ▸ Already on 'master'
INFO [2020-09-30 17:48:23.46]: ▸ Your branch is up to date with 'origin/master'.
INFO [2020-09-30 17:48:23.47]: 🔓  Decrypted '5WX395C8CZ.p12'
INFO [2020-09-30 17:48:23.47]: 🔓  Decrypted '5WX395C8CZ.cer'
INFO [2020-09-30 17:48:23.47]: 🔓  Successfully decrypted certificates repo
INFO [2020-09-30 17:48:23.55]: Verifying that the certificate and profile are still valid on the Dev Portal...
Reading keychain entry, because either user or password were empty
Loading session from '/Users/macbookair/.fastlane/spaceship/acc@flipdish.ie/cookie'
Available session is not valid any more. Continuing with normal login.
Multiple teams found on the Developer Portal, please enter the number of the team you want to use: 
1) D3FG4... "Prandium Limited" (Company/Organization)
2) D8FM... "Prandium Limited Ent" (In-House)
2
WARN [2020-09-30 17:48:30.92]: To not be asked about this value, you can specify it using 'app_identifier'
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/highline-1.7.10/lib/highline.rb:624: warning: Using the last argument as keyword parameters is deprecated
 [2020-09-30 17:48:30.92]: The bundle identifier(s) of your app (comma-separated): 
WARN [2020-09-30 17:48:33.29]: Couldn't find a valid code signing identity for enterprise... creating one for you now
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------
Successfully loaded Appfile at path '/Users/macbookair/Desktop/Code/ios/fastlane/Appfile'
-------

+----------------------+----------------------------------------------------------+
|                            Summary for cert 2.161.0                             |
+----------------------+----------------------------------------------------------+
| platform             | ios                                                      |
| development          | false                                                    |
| generate_apple_certs | true                                                     |
| force                | true                                                     |
| username             | acc@flipdish.ie                                        |
| keychain_path        | /Users/macbookair/Library/Keychains/flipdish.keychain-db |
| keychain_password    | ********                                                 |
+----------------------+----------------------------------------------------------+

INFO [2020-09-30 17:48:33.40]: Starting login with user 'acc@flipdish.ie'
Reading keychain entry, because either user or password were empty
Loading session from '/Users/macbookair/.fastlane/spaceship/acc@flipdish.ie/cookie'
Available session is not valid any more. Continuing with normal login.
Multiple teams found on the Developer Portal, please enter the number of the team you want to use: 
1) D3FG4... "Prandium Limited" (Company/Organization)
2) D8FM... "Prandium Limited Ent" (In-House)
2
INFO [2020-09-30 17:48:48.24]: Successfully logged in
INFO [2020-09-30 17:48:50.02]: $ security import /var/folders/t7/pjy501d53hzg38d68vdnyyrh0000gn/T/d20200930-11040-1548rs4/certs/enterprise/4ZHVMCWM55.p12 -k '/Users/macbookair/Library/Keychains/flipdish.keychain-db' -P '' -T /usr/bin/codesign -T /usr/bin/security -T /usr/bin/productbuild
INFO [2020-09-30 17:48:59.75]: ▸ 1 key imported.
[⠋] Setting key partition list... (this can take a minute if there are a lot of keys installed) INFO [2020-09-30 17:48:59.77]: $ security set-key-partition-list -S apple-tool:,apple: -s -k flipdish /Users/macbookair/Library/Keychains/flipdish.keychain-db 1> /dev/null
[✔] Setting key partition list... (this can take a minute if there are a lot of keys installed) 
INFO [2020-09-30 17:48:59.96]: $ security import /var/folders/t7/pjy501d53hzg38d68vdnyyrh0000gn/T/d20200930-11040-1548rs4/certs/enterprise/4ZHVMCWM55.cer -k '/Users/macbookair/Library/Keychains/flipdish.keychain-db' -P '' -T /usr/bin/codesign -T /usr/bin/security -T /usr/bin/productbuild
INFO [2020-09-30 17:49:00.04]: ▸ 1 certificate imported.
[⠋] Setting key partition list... (this can take a minute if there are a lot of keys installed) INFO [2020-09-30 17:49:00.07]: $ security set-key-partition-list -S apple-tool:,apple: -s -k flipdish /Users/macbookair/Library/Keychains/flipdish.keychain-db 1> /dev/null
[✔] Setting key partition list... (this can take a minute if there are a lot of keys installed) 
INFO [2020-09-30 17:49:00.44]: Successfully generated 4ZHVMCWM55 which was imported to the local machine.
INFO [2020-09-30 17:49:00.44]: $ security list-keychains -d user
INFO [2020-09-30 17:49:00.46]: ▸     "/Users/macbookair/Library/Keychains/login.keychain-db"
INFO [2020-09-30 17:49:00.46]: ▸     "/Users/macbookair/Library/Keychains/flipdish.keychain-db"
INFO [2020-09-30 17:49:00.46]: $ security find-certificate -c 'Apple Worldwide Developer Relations Certification Authority' /Users/macbookair/Library/Keychains/login.keychain-db
INFO [2020-09-30 17:49:00.50]: ▸ keychain: "/Users/macbookair/Library/Keychains/login.keychain-db"
INFO [2020-09-30 17:49:00.50]: ▸ version: 512
INFO [2020-09-30 17:49:00.50]: ▸ class: 0x80001000 
INFO [2020-09-30 17:49:00.50]: ▸ attributes:
INFO [2020-09-30 17:49:00.50]: ▸     "alis"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2020-09-30 17:49:00.50]: ▸     "cenc"<uint32>=0x00000003 
INFO [2020-09-30 17:49:00.51]: ▸     "ctyp"<uint32>=0x00000001 
INFO [2020-09-30 17:49:00.51]: ▸     "hpky"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2020-09-30 17:49:00.51]: ▸     "issu"<blob>=0x3062310B300906035504061302555331133011060355040A130A4150504C4520494E432E31263024060355040B131D4150504C452043455254494649434154494F4E20415554484F52495459311630140603550403130D4150504C4520524F4F54204341  "0b1\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\023\012APPLE INC.1&0$\006\003U\004\013\023\035APPLE CERTIFICATION AUTHORITY1\0260\024\006\003U\004\003\023\015APPLE ROOT CA"
INFO [2020-09-30 17:49:00.51]: ▸     "labl"<blob>="Apple Worldwide Developer Relations Certification Authority"
INFO [2020-09-30 17:49:00.51]: ▸     "skid"<blob>=0x88271709A9B618608BECEBBAF64759C55254A3B7  "\210'\027\011\251\266\030`\213\354\353\272\366GY\305RT\243\267"
INFO [2020-09-30 17:49:00.51]: ▸     "snbr"<blob>=0x01DEBCC4396DA010  "\001\336\274\3049m\240\020"
INFO [2020-09-30 17:49:00.51]: ▸     "subj"<blob>=0x308196310B300906035504061302555331133011060355040A0C0A4170706C6520496E632E312C302A060355040B0C234170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E733144304206035504030C3B4170706C6520576F726C647769646520446576656C6F7065722052656C6174696F6E732043657274696669636174696F6E20417574686F72697479  "0\201\2261\0130\011\006\003U\004\006\023\002US1\0230\021\006\003U\004\012\014\012Apple Inc.1,0*\006\003U\004\013\014#Apple Worldwide Developer Relations1D0B\006\003U\004\003\014;Apple Worldwide Developer Relations Certification Authority"
INFO [2020-09-30 17:49:03.82]: Verifying the certificate is properly installed locally...
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/interface.rb:141: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/errors/fastlane_error.rb:9: warning: The called method `initialize' is defined here

Looking for related GitHub issues on fastlane/fastlane...
Search query: Could not find the newly generated certificate installed

URL: https://api.github.com/search/issues?q=Could%20not%20find%20the%20newly%20generated%20certificate%20installed+repo:fastlane/fastlane
➡️  [deliver] Spaceship::UnexpectedResponse: An attribute value is not acceptable for the current resource state.
    https://github.com/fastlane/fastlane/issues/17307 [open] 5 💬
    2 hours ago

➡️  fastlane does not search the specified keychain for a new certificate
    https://github.com/fastlane/fastlane/issues/16321 [closed] 2 💬
    27 Jul 2020

➡️  [match] Generate profiles with new Xcode 11 certificates attached
    https://github.com/fastlane/fastlane/issues/15031 [closed] 44 💬
    20 May 2020

and 32 more at: https://github.com/fastlane/fastlane/search?q=Could%20not%20find%20the%20newly%20generated%20certificate%20installed&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
Traceback (most recent call last):
	17: from /usr/local/Cellar/fastlane/2.161.0/libexec/bin/fastlane:23:in `<main>'
	16: from /usr/local/Cellar/fastlane/2.161.0/libexec/bin/fastlane:23:in `load'
	15: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/bin/fastlane:23:in `<top (required)>'
	14: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane/lib/fastlane/cli_tools_distributor.rb:111:in `take_off'
	13: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/commands_generator.rb:25:in `start'
	12: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/commands_generator.rb:187:in `run'
	11: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	10: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	 9: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	 8: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	 7: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	 6: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/commands_generator.rb:68:in `block (3 levels) in run'
	 5: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/runner.rb:94:in `run'
	 4: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/runner.rb:161:in `fetch_certificate'
	 3: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/generator.rb:39:in `generate_certificate'
	 2: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/cert/lib/cert/runner.rb:17:in `launch'
	 1: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/interface.rb:141:in `user_error!': Could not find the newly generated certificate installed (FastlaneCore::Interface::FastlaneError)
	17: from /usr/local/Cellar/fastlane/2.161.0/libexec/bin/fastlane:23:in `<main>'
	16: from /usr/local/Cellar/fastlane/2.161.0/libexec/bin/fastlane:23:in `load'
	15: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/bin/fastlane:23:in `<top (required)>'
	14: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane/lib/fastlane/cli_tools_distributor.rb:111:in `take_off'
	13: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/commands_generator.rb:25:in `start'
	12: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/commands_generator.rb:187:in `run'
	11: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	10: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	 9: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	 8: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	 7: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	 6: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/commands_generator.rb:68:in `block (3 levels) in run'
	 5: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/runner.rb:94:in `run'
	 4: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/runner.rb:161:in `fetch_certificate'
	 3: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/match/lib/match/generator.rb:39:in `generate_certificate'
	 2: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/cert/lib/cert/runner.rb:17:in `launch'
	 1: from /usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/fastlane-2.161.0/fastlane_core/lib/fastlane_core/ui/interface.rb:141:in `user_error!': \e[31m[!] Could not find the newly generated certificate installed\e[0m (FastlaneCore::Interface::FastlaneError) </pre>
</details>

### Environment

<details>
  <pre> [✔] 🚀 
[18:07:18]: fastlane detected a Gemfile in the current directory
[18:07:18]: However, it seems like you didn't use `bundle exec`
[18:07:18]: To launch fastlane faster, please use
[18:07:18]: 
[18:07:18]: $ bundle exec fastlane env
[18:07:18]: 
[18:07:18]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
[18:07:24]: Generating fastlane environment output, this might take a few seconds...
Your Gemfile lists the gem fastlane-plugin-sentry (>= 0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once.
While it's not a problem now, it could cause errors if you change the version of one of them later.
[18:07:26]: Installing Ruby gem 'fastlane-plugin-sentry'...
[18:07:34]: Successfully installed 'fastlane-plugin-sentry'
[18:07:34]: Plugin 'Sentry' overwrites already loaded action 'sentry_upload_dsym'
[18:07:34]: Plugin 'Sentry' overwrites already loaded action 'sentry_set_commits'
[18:07:34]: Plugin 'Sentry' overwrites already loaded action 'sentry_upload_sourcemap'
[18:07:34]: Plugin 'Sentry' overwrites already loaded action 'sentry_finalize_release'
[18:07:34]: Plugin 'Sentry' overwrites already loaded action 'sentry_upload_file'
[18:07:34]: Plugin 'Sentry' overwrites already loaded action 'sentry_upload_proguard'
[18:07:34]: Plugin 'Sentry' overwrites already loaded action 'sentry_create_release'
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                   |
| --------------------------- | ------------------------------------------------------- |
| OS                          | 10.15.6                                                 |
| Ruby                        | 2.7.1                                                   |
| Bundler?                    | false                                                   |
| Git                         | git version 2.24.3 (Apple Git-128)                      |
| Installation Source         | /usr/local/Cellar/fastlane/2.161.0/libexec/bin/fastlane |
| Host                        | Mac OS X 10.15.6 (19G2021)                              |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.7.1_2/lib                      |
| OpenSSL Version             | OpenSSL 1.1.1f  31 Mar 2020                             |
| Is contained                | false                                                   |
| Is homebrew                 | false                                                   |
| Is installed via Fabric.app | false                                                   |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/             |
| Xcode Version               | 12.0.1                                                  |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.161.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                 | Version | Update-Status |
| ---------------------- | ------- | ------------- |
| fastlane-plugin-sentry | 1.6.0   | ✅ Up-To-Date  |
| fastlane-plugin-sentry | 1.6.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.4.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| CFPropertyList            | 3.0.2        |
| claide                    | 1.0.3        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.3.0        |
| xcodeproj                 | 1.18.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| public_suffix             | 4.0.6        |
| addressable               | 2.7.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.8.1        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| babosa                    | 1.0.3        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.76.0       |
| faraday                   | 1.0.1        |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.7        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.2.0        |
| gh_inspector              | 1.1.3        |
| json                      | 2.3.0        |
| mini_magick               | 4.10.1       |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.7.6        |
| bundler                   | 2.1.2        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| jwt                       | 2.2.2        |
| uber                      | 0.1.0        |
| declarative               | 0.0.20       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mini_mime                 | 1.0.2        |
| multi_json                | 1.15.0       |
| signet                    | 0.14.0       |
| memoist                   | 0.16.2       |
| os                        | 1.1.1        |
| googleauth                | 0.13.1       |
| httpclient                | 2.8.3        |
| google-api-client         | 0.38.0       |
| google-cloud-env          | 1.3.3        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-core         | 1.5.0        |
| rake                      | 13.0.1       |
| digest-crc                | 0.6.1        |
| google-cloud-storage      | 1.29.0       |
| emoji_regex               | 3.0.0        |
| jmespath                  | 1.4.0        |
| aws-partitions            | 1.376.0      |
| aws-eventstream           | 1.1.0        |
| aws-sigv4                 | 1.2.2        |
| aws-sdk-core              | 3.108.0      |
| aws-sdk-kms               | 1.38.0       |
| aws-sdk-s3                | 1.81.1       |
| uri                       | 0.10.0       |
| forwardable               | 1.3.1        |
| logger                    | 1.4.2        |
| cgi                       | 0.1.0        |
| timeout                   | 0.1.0        |
| stringio                  | 0.1.0        |
| ipaddr                    | 1.2.2        |
| openssl                   | 2.1.2        |
| ostruct                   | 0.2.0        |
| strscan                   | 1.0.3        |
| date                      | 3.0.0        |
| delegate                  | 0.1.0        |
| fileutils                 | 1.4.1        |
| io-console                | 0.5.6        |
| zlib                      | 1.1.0        |
| singleton                 | 0.1.0        |
| rexml                     | 3.2.3        |
| open3                     | 0.1.0        |
| yaml                      | 0.1.0        |
| psych                     | 3.1.0        |
| mutex_m                   | 0.1.0        |
| webrick                   | 1.6.0        |
| etc                       | 1.1.0        |
| fastlane-plugin-sentry    | 1.6.0        |
</details>

*generated on:* **2020-09-30**
</details>
[18:07:35]: Take notice that this output may contain sensitive information, or simply information that you don't want to make public.
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/highline-1.7.10/lib/highline.rb:624: warning: Using the last argument as keyword parameters is deprecated
[18:07:35]: 🙄  Wow, that's a lot of markdown text... should fastlane put it into your clipboard, so you can easily paste it on GitHub? (y/n)
/usr/local/Cellar/fastlane/2.161.0/libexec/gems/highline-1.7.10/lib/highline.rb:624: warning: Using the last argument as keyword parameters is deprecated
 y
[18:07:46]: Successfully copied markdown into your clipboard 🎨
[18:07:46]: Open https://github.com/fastlane/fastlane/issues/new to submit a new issue ✅
</pre>
</details>

**Comment by arsolisGen** [comment]
I have the same problem, in my keychain the certificate appears as "certificate is not trusted" but i see that the certificate is created in the apple dashboard and seems to be ok. Fastlane doen't push anything to the repository.

**Comment by arsolisGen** [comment]
I solve the issue reinstalling apple intermediate certificates. You can download it here:

https://developer.apple.com/certificationauthority/AppleWWDRCA.cer
https://www.apple.com/certificateauthority/AppleWWDRCAG3.cer

Hope it helps!

**Comment by katanaBoy** [comment]
Oh thank you! Will try this too.

**Comment by katanaBoy** [comment]
@arsolisGen Thank you soo much you are a life saver!!  That solved the problem

**Comment by taschik** [comment]
Thanks @arsolisGen for the solution. This also works if you get the error that I got:

```
Warning: unable to build chain to self-signed root for signer "iPhone Developer: name (xxxxxxxxxx)" 
```

**Comment by rogerluan** [comment]
I just found this https://developer.apple.com/support/expiration/

Maybe you guys had the certificates that were issued before September 1st 2020, and thus got expired when these new ones came out (maybe?) 

Could anyone share what the process of "reinstalling apple intermediate certificates" looks like? Is it just downloading those 2 certs and putting them into your keychain? I'm asking because we could improve this process by creating a new feature in _match_ that will reinstall those for you 😊

**Comment by katanaBoy** [comment]
@rogerluan Hey, that would be awesome. Yes that was the solution for me. On the CI I just use few commands to add it in the keychain.

**Comment by arsolisGen** [comment]
yes, that would be great, thank you!

**Comment by taschik** [comment]
I have added the certificate to the repository and adding it to the keychain on CI builds like this:

```
# Importing Apples intermediate WWDR certificate to ensure a trusted chain
# for enterprise distribution certificates can be established

import_certificate(
  certificate_path: './fastlane/certs/WWDRCA.cer', 
  keychain_name: ENV["CI_KEYCHAIN_NAME"],
  keychain_password: ENV["CI_KEYCHAIN_PASSWORD"]
)
```

this way you can make sure you always have the certificates needed to build up the chain. I hope this helps.

**Comment by rogerluan** [comment]
Awesome, thanks for the input everyone! And thanks for the snippet @taschik 🤗 

I'll add this to my list, with a low priority. If anyone feel like contributing, all PRs are welcome! Please tag me in your PR if you end up opening one 😊

**Comment by rogerluan** [comment]
AFAICT this was fixed here https://github.com/fastlane/fastlane/pull/17538 and just released as part of [2.167.0](https://github.com/fastlane/fastlane/releases/tag/2.167.0) 🚀 

Closing this - let me know if the latest version of fastlane doesn't fix this issue for any of you 🤗
